pkg/deploy/helm/matrix_tests/README.md

```diff
@@ -0,0 +1,133 @@
+Matrix Tests
+============
+Matrix tests are used to validate all variations of helm values.
+
+## Problem
+Sometimes user set helm values in CI deploy job with `werf deploy --set` command.
+
+Gitlab example:
+```yaml
+Deploy Production:
+  stage: tests
+  script:
+    - type multiwerf && source <(multiwerf use 1.0 beta)
+    - type werf && source <(werf ci-env gitlab --tagging-strategy tag-or-branch --verbose)
+    - werf deploy --stages-storage :local
+      --namespace ${CI_ENVIRONMENT_SLUG}
+      --set "global.env=prod"
```

Comment:
Need to change `--set` example, because `global.env` is set automatically by werf itself.

Reply:
There is parameter of werf deploy named `--env`, that used to set `global.env` and generate helm release name and kubernetes namespace. In gitlab however it is not specified for the `werf deploy` command because `werf ci-env` command sets this parameter implicitly using `WERF_ENV` environment variable.

Reply:
I added variables block to GitLab ci example with WERF_ENV environment variable set to `prod`.